#!/usr/bin/env python
PACKAGE = "sub_vision"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("lam", double_t, 0, "Lambda gain for interaction matrix",0,0,50)
gen.add("xThreshold", double_t, 0, "X tolerable error for alignment",0,0,50)
gen.add("yThreshold", double_t, 0, "Y tolerable error for alignment",0,0,50)

exit(gen.generate(PACKAGE, "visual_servo", "Servoing"))
